### DISCUSSION TITLE
Discrepancies between versions (v2 vs v3)

### DISCUSSION THREAD

**Discussion by mariabrbz** [adobe/react-spectrum#2106] [Q&A] [answered]
I have a project that uses v2.27.0 and I am trying to incorporate elements found in v3. 

Could the two versions be used in parallel given the different namespaces (@react and @adobe respectively)? If not, is there a straight forward way of upgrading to v3? 

It seems like there are a lot of discrepancies between the two versions, but also some gaps in the documentation. For example, when using v2.27.0, I can find and use components that do not actually appear in the v2.27.0 packages.

I would really appreciate it if someone could shed some light on this!

**Comment by snowystinger** [comment, marked as answer]
Hey! can you please bring this up internally? We can help you more there.
